qase-python-commons/src/qase/commons/models/base.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+from pathlib import PosixPath
+
+
+class Base:
+    def to_json(self) -> str:
```

Comment:
Multiline logs are nice to read. However, they can break when we have parallel runs where multiple loggers write simultaneously.